src/host/alias.cpp

```diff
@@ -147,9 +147,9 @@ HRESULT GetConsoleAliasWImplHelper(const std::wstring_view source,
     // Ensure output variables are initialized
     writtenOrNeeded = 0;
 
-    if (target.has_value() && target.value().size() > 0)
+    if (target.has_value() && target->size() > 0)

```

Comment:
alternatively, using empty may be clearer and more concise:

!target->empty()

Reply:
Note this also fulfills one of the ISO C++ Guidelines:
*T.143: Don’t write unintentionally nongeneric code*
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#t143-dont-write-unintentionally-nongeneric-code

Reply:
This is something outside of the scope of std::optional.  I might file another PR if this one gets merged (I won't file one now since the changes would conflict)

Reply:
Generally, if I'm touching the line anyway, I will personally often include such changes.  However, it's also reasonable to want to separate general cleanup from a more targeted change.  So fair enough.